src/EventExport/Format/HTML/templates/export.map.activities.html.twig

```diff
@@ -70,7 +70,11 @@
                             <i class="fa fa-map-marker"></i>&nbsp;
                         </td>
                         <td>
-                            {{ event.address.name }}, {{ event.address.street }}, {{ event.address.postcode }} {{ event.address.municipality }}
+                            {{ event.address.name }}
+                            {%-  if not event.address.isDummyAddress -%}
```

Comment:
Curious what the reason behind these custom twig delimiters is 🤔

Reply:
It's to remove the whitespace in front and/or behind the opening/closing braces.
In this case it's not that important but in `export.tips.html.twig` I had to use them to keep the structure of the outputted HTML the same as before since there's some unit tests that compare it to HTML files in the tests folder.